azure-pipelines.yml

```diff
@@ -22,7 +22,8 @@ steps:
   displayName: 'guardian export'
```

Comment:
What is this pipeline doing anymore?

Reply:
It was a thing that @sandersn added for some compliance thing

Reply:
My memory is that it's a component governance used for checking for supply-chain attacks. I can't remember if it ends up also checking DT, but my memory is that it does because of the way we run our tests. Useful to have running.